12LessonTwelve/in-class-project/index.html

```diff
@@ -1,10 +1,34 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Delivery Food</title>
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+  <header>
+<div id="main">
+  <h1>Order Takeaway or Delivery Food</h1>
+  <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.</p>
+</div>
+  </header>
+  <div class="row">
+    <div class="icon_home_1 col-xs-3"><img src="icon_home_1.svg"</div>

```

Comment:
you got the columns sizes incorrect
for:
* extra small you need columns size 12
* small column size 6
* large column size 3